examples/tutorials/advanced/Federated Dataset.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1>Federated Datasets</h1>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how you can develop your custom datasets that work with federated data loaders. The regular torch datasets do not always work directly with federated data loaders.The tutorial aims to highlight the differences and shows you how you could develop a federated dataset with your data that works with the federated data loader. Further you could also use Syft's Base dataset feature which simplify creating federated datasets.To demonstrate these we will show you how to load the SVHN (Street View House Numbers) dataset and convert it to a federated dataset.\n",
+    "\n",
+    "Hrishikesh Kamath - GitHub: @<a href=\"http://github.com/kamathhrishi\">kamathhrishi</a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from torch.utils.data import Dataset\n",
+    "import syft as sy  \n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#Define Syft workers Bob and Alice for federated learning\n",
+    "\n",
+    "hook = sy.TorchHook(torch)  # <-- NEW: hook PyTorch ie add extra functionalities to support Federated Learning\n",
+    "bob = sy.VirtualWorker(hook, id=\"bob\")  # <-- NEW: define remote worker bob\n",
+    "alice = sy.VirtualWorker(hook, id=\"alice\")  # <-- NEW: and alice"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "class Arguments():\n",
+    "    def __init__(self):\n",
+    "        self.batch_size = 1\n",
```

Comment:
You don't need all these like the .lr I think

Reply:
yeah, I guess its unnecessary.